src/main/java/module-info.java

```diff
@@ -38,6 +41,7 @@
 
     // Logging
     requires org.slf4j;
+    requires jul.to.slf4j;
```

Comment:
Do we really need this jul adapter?

Reply:
Yes. The Grizzly http server uses java-util-logging. To be able to handle the log output with our logging framework tinylog, we need jul-slf4j-tinylog. 

More detailed:

- We opted for SLF4J as the JabRef logging framework. We decided against direct calls to tinylog. 
- For log output of an application, there should be one logging framework
- When using the http server, JabRef internal methods are used. They might log. Thus, we have logs using of SLF4J
- Grizzly uses Java-Util-Logging - and outputs are produced.

The alternative can be to disable logging of Grizzly completely and bring it back (with the bridge) if needed.